src/components/Api.js

```diff
@@ -19,6 +19,7 @@ export default class Api {
     return fetch(`${this._baseUrl}/users/me`, {
       headers: this._headers
     })
+    .catch (err => console.log(err))
```

Comment:
` .catch (err => console.log(err))` is not needed in `Api`. Please, delete it

It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very end of any request and this end is in `index.js`  after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.